CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeService jokeService;
+        private IJokeOutput jokeOutput;
+
+        public IJokeService JokeService { get => jokeService; private set => jokeService = value??throw new ArgumentNullException(); }
```

Comment:
When throwing `ArgumentException` or `ArgumentNullException` you always want to specify the argument name.
```suggestion
        public IJokeService JokeService { get => jokeService; private set => jokeService = value??throw new ArgumentNullException(nameof(value)); }
```